### PR TITLE
allow View.redraw() redraw defx buffer when not current buffer, avoid window jump

### PR DESCRIPTION
**Pull request by zgpio**
Redraw function should not limit the current defx buffer.

### CODE REVIEW

**Comment by Shougo** [comment]
Why this feature is needed?
Please describe to me.

File: rplugin/python3/defx/view.py [hunk 1]
```diff
@@ -154,10 +149,12 @@ def redraw(self, is_force: bool = False) -> None:
         self._buffer.options['modifiable'] = False
         self._buffer.options['modified'] = False
 
-        self._vim.call('cursor', [prev_linenr, 0])
-
-        if prev:
-            self.search_file(prev['action__path'], prev['_defx_index'])
+        # TODO: How to set cursor position for other buffer when
```
Comment (Shougo) [inline rplugin/python3/defx/view.py]: Unfortunately, nothing.
You must jump to buffer to change the position.

**Comment by zgpio** [comment]
> Why this feature is needed?
> Please describe to me.

window switch when redraw multi defx buffer, it annoy me. Please see [video](https://asciinema.org/a/VZp0GExD6mAPZJ05ASR2wOIjb).

File: rplugin/python3/defx/view.py (hunk 1 above)
Comment (zgpio) [reply rplugin/python3/defx/view.py]: I made a feature request, https://github.com/neovim/neovim/issues/10822#issue-483150362

**Comment by Shougo** [comment]
OK.

**Pull request merged**